lib/template-compiler.js

```diff
@@ -58,6 +58,12 @@ module.exports = function (html) {
       options.transformToRequire
     )
   }
+  var compilerVersion = require('vue-template-compiler/package.json').version
+  var vueVersion = require('vue').version
+
+  if (vueVersion !== compilerVersion) {
+    this.emitError(`vue version ${vueVersion} mismatch vue-template-compiler version ${compilerVersion}`)
```

Comment:
Let's use the same language found [here](https://github.com/vuejs/vue/blob/dev/packages/vue-template-compiler/index.js#L9-L14)